blog/docker-commit-convert-a-container-into-an-image.md

```diff
@@ -0,0 +1,30 @@
+# Docker commit: convert a container into an image
+
+We can convert a specific container into a base Docker image. It's not the recommended way to create images because these could include useless cached or session data. The easier way to control the data that it's stored into the Docker layers is using the `Dockerfile`.
```

Comment:
```suggestion
We can convert a specific container into a base Docker image, but that's not the recommended way to create images because these could include useless cached or session data. 
The easiest way to control the data stored in the Docker layers is using the `Dockerfile`.
```